caravel/assets/javascripts/welcome.js

```diff
@@ -4,12 +4,13 @@ var jQuery = window.jQuery = $;
 require('../stylesheets/caravel.css');
 require('../stylesheets/welcome.css');
 require('bootstrap');
-require('datatables');
+require('datatables.net-bs');
```

Comment:
oh interesting, if you don't have a fresh build, it could have been using old node_modules, `npm prune` cleans up to whatever is defined in the `package.json`